ec-gpu-gen/src/lib.rs

```diff
@@ -219,6 +206,10 @@ where
     .replace("FIELD", name)
 }
 
+pub fn common() -> String {
```

Comment:
cany you add some docs about this please?

Reply:
Done.